examples/pagerank.cpp

```diff
@@ -0,0 +1,164 @@
+// Copyright 2016 Husky Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost/tokenizer.hpp>
+#include <cmath>
+#include <string>
+#include <vector>
+#include "base/log.hpp"
+#include "core/channel/channel_factory.hpp"
+#include "core/context.hpp"
+#include "core/engine.hpp"
+#include "core/executor.hpp"
+#include "core/job_runner.hpp"
+#include "core/objlist.hpp"
+#include "io/input/hdfs_line_inputformat.hpp"
+#include "io/input/mongodb_inputformat.hpp"
+
+
+namespace husky {
+
+namespace Graph {
```

Comment:
Notice that  in `lib`, the namespace is also `lib`. So you may add `lib::Graph`.